typescript/src/detectors/OpenWerewolf_app.ts

```
@@ -0,0 +1,481 @@
+      }
+    }
+  });
+  socket.on("leaveGame", function() {
+    server.leaveGame(thisPlayerId);
+  });
+  socket.on("disconnect", function() {
+    server.removeSocketFromPlayer(thisPlayerId, socket);
+    server.kick(thisPlayerId);
+  });
+  socket.on("gameClick", function(gameId: string) {
+    if (parseInt(gameId) != NaN) {
```

Biome (2.1.2)

[error] Use the Number.isNaN function to compare with NaN.

Unsafe fix: Use Number.isNaN() instead.


(lint/correctness/useIsNan)